.sh-extension

```diff
@@ -1,9 +1,13 @@
 {
     "requiredFiles": [],
-    "buildLogs": [],
-    "appLogs": [
-        "appsody"
-    ],
+    "appWorkspaceLogs": {
+        "files": {
+            "appsody": null
```

Comment:
Since it is a key-value pair that represents, key as the suffix of the log and value as the path of the log files. We set the path to be null by default because the app log path is not concrete before runtime, i.e the app log path is something like `/codewind-workspace/.logs/<project-name-and-id>`. The project name and id is not defined till runtime so the path needs to be dynamically created and that is done part of the new logs change.

Reply:
I think you mean *prefix* of the log.  I see what you mean, thanks.